### PR TITLE
Add 008 network tests

### PR DESCRIPTION
**Pull request by rvem**
## Description
Problem: New protocol arrived.

Solution: Run network tests on 008 local-chain and don't.

Based on https://gitlab.com/morley-framework/morley-ledgers/-/merge_requests/31

## Related issue(s)

https://issues.serokell.io/issue/TM-491
#### Related changes (conditional)

- Tests
  - [ ] If I added new functionality, I added tests covering it.
  - [ ] If I fixed a bug, I added a regression test to prevent the bug from
        silently reappearing again.

- Documentation
  - [ ] I checked whether I should update the docs and did so if necessary:
    - [README](../tree/master/README.md)
    - Haddock

#### Stylistic guide (mandatory)

- [ ] My commits comply with [the policy used in Serokell](https://www.notion.so/serokell/Where-and-how-to-commit-your-work-58f8973a4b3142c8abbd2e6fd5b3a08e).
- [ ] My code complies with the [style guide](../tree/master/docs/code-style.md).

### CODE REVIEW

File: .buildkite/pipeline.yml [hunk 1]
```diff
@@ -76,6 +85,20 @@ steps:
...
+    if: build.source == "schedule"
+    # use another agent for long scheduled jobs
+    agents:
+      queue: "scheduled"
+    env:
+      NETTEST_NODE_ADDR: "edo.testnet.tezos.serokell.team"
+      NETTEST_NODE_PORT: "8732"
+      # Note that testnet moneybag can run out of tz. If this happened, someone should transfer it some
+      # more tz, its address: tz1NCV82ThTCF1vU2KEGv3ZeZw4hhAPbD5UM
+      MONEYBAG: "unencrypted:edsk3UH7bwt2i3D6VQCmT8ZkMufdFEqJr79StJjguuk6W4wZccSp9J"
```
Comment (gromakovsky) [inline .buildkite/pipeline.yml:98]: Here comes my usual suggestion to use the same `MONEYBAG` everywhere and put it into env variables that will be applied to all steps.

**Review by gromakovsky — APPROVED** [review]
LGTM (don't forget that some things should be updated to `master` first).

**Comment by rvem** [comment]
We won't be able to merge it anytime soon, btw, because https://gitlab.com/morley-framework/morley-multisig/-/merge_requests/5 is waiting for the comb pairs in morley

**Review by pasqu4le — APPROVED** [review]
LGTM as well.

Needs another small bump of dependencies, including `morley-mutisig`.

**Comment by rvem** [comment]
oh well, looks like morley-ledgers should be updated first

File: .buildkite/pipeline.yml (hunk 1 above)
Comment (gromakovsky) [reply .buildkite/pipeline.yml:93]: So apparently you decided not to care about this suggestion :sob: 

Anyway, this repo is inactive, so I think it doesn't matter.

**Pull request merged**